frontend/src/components/Home/BulletPoint/BulletPoint.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import {ThemeProvider, makeStyles} from '@material-ui/core/styles';
+import AssessmentOutlinedIcon from '@material-ui/icons/AssessmentOutlined';
+import CallOutlinedIcon from '@material-ui/icons/CallOutlined';
+import ToysOutlinedIcon from '@material-ui/icons/ToysOutlined';
+import Typography from '@material-ui/core/Typography';
+import {useMediaQuery} from '@material-ui/core'
+
+const useStyles = makeStyles({
+    container: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        width: 314,
+        height: 400,
+        border: 'solid 1px lightgrey',
+        backgroundColor: 'white'
+    },
+    smallScreenContainer: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        width: 314,
+        height: 400,
+        border: 'solid 1px lightgrey',
+        backgroundColor: 'white',
+        margin: 8,
+    },
+    alignTitle: {
+        display: 'flex',
+        margin: 24
+    },
+    customizeTitle: {
+        fontWeight: 'bold',
+        marginLeft: 4
+    },
+    customizeIcon: {
+        marginRight: 4
+    },
+    customizeText: {
+        paddingLeft: 16,
+        paddingRight: 16,
+        marginBottom: 16
+    }
+})
+
+const BulletPoint = (props) => {
+    const classes = useStyles()
+    const smallScreen = useMediaQuery("(max-width: 950px)")
```

Comment:
semicolon. Won't comment anymore in this PR but seriously add them please everywhere.